app/utils/parse-base64.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Validates a file format by analyzing the data label (the first 15 characters of a base64 string)
+ * @param {string} base64 - base64 string
+ * @returns {string|Error} - validated file format or Error object
```

Comment:
See comment above about explicitly returning an error. If you have to return error information, do it as an object literal or a class. Something that's well-defined like guaranteeing there's a `msg` property